css/style.css

```diff
@@ -633,3 +631,356 @@ input[type="submit"]:active {
+@media screen and (min-width: 768px) {
+  #mobile-design {
+    width: 100%;
+    height: 4561px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navlinks {
+    display: flex;
+    width: 300px;
+    margin-right: 10%;
+    font-family: Inter, sans-serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    align-items: center;
+  }
+
+  .links a {
+    color: #fff;
+    margin-right: 25px;
+    text-decoration-line: none;
+  }
+
+  /* BEGIN Headline Section */
+  .headline-desktop {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 798px;
+    background: #1c1a19;
+    background-image: url("/images/illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: 100%;
+  }
+
+  .headline-info {
+    width: 566px;
+    height: 739px;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+  }
+
+  .headline-info h2 {
+    margin: 0;
+    font-family: Inter, sans-serif;
+    font-size: 2rem;
+    font-weight: 500;
+    animation-name: text;
+    animation-duration: 3s;
+    animation-iteration-count: infinite;
+  }
+
+  .lateral-bar {
+    width: 15%;
+    display: flex;
+    align-items: center;
+    order: -1;
+  }
+
+  .social-icons {
+    margin-left: 15px;
+    margin-top: 279px;
+    width: 200px;
+    height: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .support-text-desktop {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #fff;
+    margin: 12px 24px 0 20px;
+    gap: 24px;
+    animation-duration: 3s;
+    animation-name: slidein;
+  }
+
+  /* END Headline Section */
+
+  /* BEGIN Work Section */
+  .works-section-desktop {
+    width: 100%;
+    height: 1810px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .vector-works {
+    display: none;
+  }
+
+  .works-head {
+    margin-top: 135px;
+    margin-bottom: 114px;
+    display: flex;
+    margin-left: 141px;
+    align-items: center;
+  }
+
+  .vector-works-desktop {
+    display: block;
+    margin-top: 0.5em;
+    margin-bottom: 0.5em;
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+  }
+
+  .stories-desktop {
+    width: 1154px;
+    height: 448px;
+    margin-left: 141px;
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .stories-desktop img {
+    width: 684px;
+    height: 448px;
+    margin-left: 0;
+  }
+
+  .stories-desktop div {
+    height: 348px;
+    width: 448px;
+    left: 708px;
+    top: 0;
+    border-radius: 0;
+  }
+
+  .works-info-desktop {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .works-cards-desktop {
+    margin-left: 141px;
+    margin-top: 0;
+    width: 1156px;
+    height: 907px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 1rem;
+  }
+
+  .cards-work-desktop {
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.
-   The project text should take up no more than 3 lines of text. Your project text currently takes up 4 lines of text.

| Your Design  |Template Design |
| ------------- | ------------- |
| <img width="918" alt="image" src="https://user-images.githubusercontent.com/52934128/228739647-0d90c333-965f-44c8-a988-fa145c5c7ebc.png"> | ![image](https://user-images.githubusercontent.com/52934128/178112550-f1b33dc5-838e-4251-898a-ea299dc20431.png) |

Reply:
Hello @topeogunleye, thanks for your time and suggestion, please, would you one give advice to making this requirement?, I was trying with some options but no working, I tried with -webkit-text-fill-color and color:transparent but are not ok.